selenium_webdriver/src/main/java/core/driver/DriverSingleton.java

```diff
@@ -0,0 +1,33 @@
+package core.driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class DriverSingleton {
+
+    private static WebDriver driver;
+    private static final String browserName = "chrome";
+
+    public static WebDriver driverInitialize() {
+        if (driver == null) {
+            if (browserName.equalsIgnoreCase("chrome")) {
+                driver = new ChromeDriver();
+
+            } else if (browserName.equalsIgnoreCase("firefox")) {
```

Comment:
unreachable conditions because 'browserName' is always "chrome". Moreover, you don't need anymore